routes/api/browse/read-browse.js

```diff
@@ -7,5 +7,24 @@ var router = express.Router();
 
 console.log('read-browse');
 
+
+app.get("/api/requests", function(req, res) {
+    // 1. Add a join to include all of users
+    db.User.findAll({}).then(function(dbRequest) {
+      res.json(dbRequest);
+    });
+  });
+
+  app.get("/api/requests/:id", function(req, res) {
+    // 2; Add a join to include all of users
+    db.Author.findOne({
+      where: {
+        id: req.params.id
+      }
+    }).then(function(dbRequest) {
+      res.json(dbRequest);
+    });
+  });
```

Comment:
We can delete these entries for now since we haven't finalized how the browse API will work